requirements.txt

```diff
@@ -1,8 +1,9 @@
 absl-py==0.9.0
 astor==0.8.1
 cachetools==4.0.0
-certifi==2019.11.28
+certifi==2020.6.20
```

Comment:
While you are at
While you are at updating requirements, please switch to these latest versions which work with Python 3.8:


```diff
diff --git a/requirements.txt b/requirements.txt
index 29178bc..45aafb4 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,7 +3,7 @@ astor==0.8.1
 cachetools==4.0.0
 certifi==2019.11.28
 chardet==3.0.4
-gast==0.2.2
+gast==0.3.3
 google-auth==1.11.3
 google-auth-oauthlib==0.4.1
 google-pasta==0.2.0
@@ -13,7 +13,7 @@ idna==2.9
 joblib==0.14.1
 Keras==2.3.1
 Keras-Applications==1.0.8
-Keras-Preprocessing==1.1.0
+Keras-Preprocessing==1.1.2
 Markdown==3.2.1
 numpy==1.18.2
 oauthlib==3.1.0
@@ -29,9 +29,9 @@ rsa==4.0
 scikit-learn==0.22.2.post1
 scipy==1.4.1
 six==1.14.0
-tensorboard==2.1.1
-tensorflow==2.1.0
-tensorflow-estimator==2.1.0
+tensorboard==2.3.0
+tensorflow==2.3.0
+tensorflow-estimator==2.3.0
 termcolor==1.1.0
 urllib3==1.25.8
 Werkzeug==1.0.0
```

Reply:
Noted, will update.